conf/script/build-cortx-ha

```diff
@@ -19,8 +19,21 @@
 set -eu -o pipefail
 
 HA_PATH="<HA_PATH>"
+HA_LOG="/var/log/seagate/cortx/ha"
+
+configure_alert() {
+    pcs alert show | grep ha_iem_event || pcs alert create id=ha_iem_event \
```

Comment:
Add a validation that pacemaker cluster is up.

Reply:
Done
Added
```
    # Check pacemaker cluster
    pcs status || { echo "cluster not running"; exit 1;}
```